src/test/moment/is_between.js

```diff
@@ -72,6 +72,38 @@ test('is between without units', function (assert) {
     assert.equal(+m, +mCopy, 'isBetween second should not change moment');
 });
 
+test('is between without units inclusive', function (assert) {
+    var m = moment(new Date(2011, 3, 2, 3, 4, 5, 10)), mCopy = moment(m);
+    assert.equal(m.isBetween(
+        moment(new Date(2011, 3, 2, 3, 4, 5, 10)),
+        moment(new Date(2012, 3, 2, 3, 4, 5, 10)), {'inclusive':'()'}), false, 'between, start and end are excluded');
+    assert.equal(m.isBetween(
+        moment(new Date(2011, 3, 2, 3, 4, 5, 10)),
+        moment(new Date(2012, 3, 2, 3, 4, 5, 10)), {'inclusive':'(]'}), false, 'between, start is excluded and end is included should fail on same start date');
```

Comment:
Agreed.  If one side or the other is excluded, and the values are the same, then it has zero duration and thus no values are in the range at all, so `false` is correct.